app/content/models.py

```diff
@@ -1,12 +1,9 @@
 from django.db import models
-from django.utils import timezone
-from django.contrib.postgres.fields import ArrayField
 
 
 class BaseModel(models.Model):
-    id = models.PositiveIntegerField(primary_key=True)
     title = models.CharField(max_length=280)
-    creation_date = models.DateTimeField(default=timezone.now)
+    creation_date = models.DateTimeField(auto_now_add=True)
```

Comment:
Is this the same as `default=timezone.now`?

Reply:
Yep, it's the exact same thing, but in this case doing one (explicit) import less.